server/resources/user_login.py

```diff
@@ -32,8 +33,7 @@ def post(self):
                 'user_id': user.id,
                 'full_name': user.full_name,
                 'email': user.email,
-                'experience': user.experience
-
+                'experience': user_experience
```

Comment:
Don't we need to send this every time we need to get the user? I would make a user model method to get the user with the experience and it should be one query tbh

Reply:
Isn't there a route to get the user when they first open the app? That should also be sending the experience as well

Reply:
added a UserModel method to get user along with experience. changed user_login and user_get to user this method instead.